benchmarks/meshoptimizer/build-native.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Build all of the shootout benchmarks as native shared libraries (Linux-only).
+#
+# Usage: ./build-native.sh
+
+(set -x; rm -r -f meshoptimizer);
```

Comment:
If we're going to print every command we might as well just `set -x` once at the top.